src/features/dataset/DatasetScrollLayout.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { useParams } from 'react-router-dom'
+import { useInView } from 'react-intersection-observer'
+import classNames from 'classnames'
+
+import { newQriRef } from '../../qri/ref'
+import { Dataset } from '../../qri/dataset'
+import { selectDsPreview } from '../dsPreview/state/dsPreviewState'
+import { selectSessionUser } from '../session/state/sessionState'
+import { selectSessionUserCanEditDataset } from './state/datasetState'
+import DatasetHeader from './DatasetHeader'
+import DatasetMiniHeader from '../dataset/DatasetMiniHeader'
+
+interface DatasetFixedLayoutProps {
```

Comment:
should this be `DatasetScrollLayoutProps`?

Reply:
Good catch